app/models/work-item.ts

```diff
@@ -782,4 +783,16 @@ export async function computeWorkItemDurationOutlierThresholdForJobService(
 
   return threshold;
 
-}
\ No newline at end of file
+}
+
+// Listen for work items being created and put a message on the scheduler queue asking it to
+// schedule some WorkItems for the service
+eventEmitter.on(WorkItemEvent.CREATED, async (workItem: WorkItem) => {
```

Comment:
I thought we were already putting messages on the queue immediately when we created a work item?

Reply:
This is not new code, I just moved it here. This is how we were doing that (using the event emitter). I moved the code here because it seemed like a better place for it.